packages/editor-ui/src/components/NodeDetailsView.vue

```diff
@@ -177,717 +177,626 @@ import { useTelemetry } from '@/composables/useTelemetry';
 import { useI18n } from '@/composables/useI18n';
 import { storeToRefs } from 'pinia';
 
-export default defineComponent({
-	name: 'NodeDetailsView',
-	components: {
-		NodeSettings,
-		InputPanel,
-		OutputPanel,
-		NDVDraggablePanels,
-		TriggerPanel,
-	},
-	props: {
-		readOnly: {
-			type: Boolean,
-		},
-		renaming: {
-			type: Boolean,
-		},
```

Comment:
since `required` is `false` by default in the old style, this should have been converted to 

```typescript
const props = withDefaults(
	defineProps<{
		readOnly?: boolean;
		renaming?: boolean;
		isProductionExecutionPreview?: boolean;
	}>(),
	{
		isProductionExecutionPreview: false,
	},
);
```

Reply:
Nice catch! Addressed it [here](https://github.com/n8n-io/n8n/pull/9788). Thanks.